app/message/save-to-cosmos.js

```diff
@@ -0,0 +1,18 @@
+const cosmosInstance = require('../cosmos')
+const { cosmosConfig } = require('../config')
+
+const saveToCosmos = async (message) => {
+  try {
+    const { filesDatabase } = await cosmosInstance()
+    console.log('************************', cosmosConfig.filesContainer)
```

Comment:
Think we can remove this log now since we only added it in/used it for debugging.